orchagent/qosorch.cpp

```diff
@@ -1783,7 +1783,7 @@ task_process_status QosOrch::handleQueueTable(Consumer& consumer, KeyOpFieldsVal
             return task_process_status::task_invalid_entry;
         }
 
-        if(tokens[0] == gMyHostName)
+        if((tokens[0] == gMyHostName) & (tokens[1] == gMyAsicName))
```

Comment:
Shouldn't this be -
if((tokens[0] == gMyHostName) && (tokens[1] == gMyAsicName))

Reply:
It was typo and fixed it now.